blocks/community/layout/community-layout.tsx

```diff
@@ -93,14 +96,21 @@ export const CommunityLayout: FC<CommunityLayoutProps> = ({ title, ogImageName,
             />
 
             <StickyHeader>
-                <TopMenu
-                    homeUrl={COMMUNITY_URL}
-                    title={COMMUNITY_TITLE}
-                    activeIndex={activeIndex}
-                    items={items}
-                    linkHandler={linkHandler}
-                    mobileOverview={false}
-                />
+                <div className={styles.sticky}>
+                    <TopMenu
+                        className={styles.topMenu}
+                        homeUrl={COMMUNITY_URL}
+                        title={COMMUNITY_TITLE}
+                        activeIndex={activeIndex}
+                        items={items}
+                        linkHandler={linkHandler}
+                        mobileOverview={false}
+                    />
+
+                    {pathname === '/community/events/' && (
+                        <CommunityAddEvent className={styles.add} size="s"/>
+                    )}
```

Comment:
There is no reason to position absolute workaround.
TopMenu supports children that will be placed in the right corner.
```suggestion
                    <TopMenu
                        className={styles.topMenu}
                        homeUrl={COMMUNITY_URL}
                        title={COMMUNITY_TITLE}
                        activeIndex={activeIndex}
                        items={items}
                        linkHandler={linkHandler}
                        mobileOverview={false}
                    >
                    {pathname === '/community/events/' && (
                        <CommunityAddEvent className={styles.add} size="s"/>
                    )}
                    </TopMenu>
```